packages/frontend/src/screens/app/screens/home/components/tournament-card.tsx

```diff
@@ -80,35 +59,24 @@ const Image = style.div`
 `;
 
 interface TCProps {
```

Comment:
should probably be named TournamentCardProps or just Props instead of abbreviated TCProps